features/machine/machine.feature

```diff
@@ -23,6 +23,13 @@ Feature: Machine features testing
     Given evaluation of `cluster_operator('machine-api').condition(type: 'Progressing')` is stored in the :co_progressing clipboard
     Then the expression should be true> cb.co_progressing["status"]=="False"
 
+  # @author zhsun@redhat.com
+  # @case_id OCP-37706
+  @smoke
+  Scenario: baremetal clusteroperator should be disabled in any deploy that is not baremetal
```

Comment:
Baremetal clusteroperator should be disabled in any deployment that is not baremetal

Reply:
I'm not sure if this works on our UPI baremetal test, let's check when we have a chance.

Reply:
ok, will check

Reply:
Thanks, can you please update the scenario name?